provider/datagen/Cargo.toml

```diff
@@ -71,13 +71,13 @@ rayon = "1.5"
 rust-format = { version = "0.3.4", features = ["token_stream"] }
 serde = { version = "1.0", default-features = false, features = ["derive", "alloc"] }
 serde_json = { version = "1.0", default-features = false, features = ["alloc"] }
-serde-aux = { version = "2.1.1", default-features = false }
+serde-aux = { version = "4.1.2", default-features = false }
 syn = {version = "1.0", features = ["parsing"] }
 tinystr = { version = "0.7.1", path = "../../utils/tinystr", features = ["alloc", "serde", "zerovec"], default-features = false }
 toml = "0.5"
 writeable = { version = "0.5.1", path = "../../utils/writeable" }
 zerovec = { version = "0.9.2", path = "../../utils/zerovec", features = ["serde", "yoke"] }
-zip = { version = "0.5", default-features = false, features = ["deflate"] }
+zip = { version = "0.6", default-features = false, features = ["deflate"] }
```

Comment:
I'd rather not update zip to 0.6 just yet since we're trying to keep dependencies in sync with other places.

Reply:
zip 0.5 depends on time 0.1, which is the root cause of this rustsec issue

Reply:
We are not using zip with that feature enabled

Reply:
If you would like to, you can add a comment to [the dependency allowlist for zip as used by datagen](https://github.com/unicode-org/icu4x/blob/4de3bfb9b4e4527a8303ad08be39e5b138c6f244/tools/depcheck/src/allowlist.rs#L145) and mention that the list should never contain `time` unless `zip` is 0.6.

Reply:
Another thing that would be fine is using a range dep: `">=0.5.0, <0.7.0"`; basically I do not want to lose the ability to use `zip` 0.5

Reply:
`zip v0.6` is also one of the dependencies of `cached-path v0.6.1`, a range dep `>=0.5.0, <0.7.0` does not change the version resolution tho

Reply:
cached_path can also get a range, then

Reply:
It should be _possible_ to use ICU4X with zip 0.5, basically.

Again, all of this is moot for the time vuln: we're not using zip with that feature enabled

Reply:
Ah, i guess `cached-path` does, which is a bit of a bummer